dump_env/cli.py

```diff
@@ -46,7 +60,28 @@ def main() -> None:
     """
     parser = _create_parser()
     args = parser.parse_args()
-    variables = dump(args.template, args.prefix)
+
+    if args.strict is True and not args.template:
+        sys.stderr.write(STRICT_WITHOUT_TEMPLATE_MSG)
+        return
+
+    strict: Union[bool, str] = args.strict
+
+    if not strict:
+        dumper = Dumper(template=args.template, prefixes=args.prefix)
+    else:
+        strict_envs = strict.split(',') if isinstance(strict, str) else []
+        dumper = StrictDumper(
+            template=args.template,
+            prefixes=args.prefix,
+            strict_envs=strict_envs
+        )
+
+    try:
+        variables = dumper.dump()
+    except StrictException as e:
+        sys.stderr.write(str(e) + '\n')
+        return
```

Comment:
I do not like this whole part from the very beginning. What is `args.strict is True`? Why is it checked like so?  Why do we have different parameters to different classes? 

I guess we can discuss other solutions. Since the current one does not look optimal.

Reply:
Agree. But I don't see other options. I don't like my own solution =(